main.tf

```diff
@@ -255,6 +259,36 @@ module "oke" {
   }
 }
 
+#fss
+module "storage" {
+  source = "./modules/storage"
+
+  # general oci parameters
+  tenancy_id          = var.tenancy_id
+  compartment_id      = var.compartment_id
+  availability_domain = var.availability_domains["fss"]
+  label_prefix        = var.label_prefix
+
+  # FSS network information
+  subnets      = var.subnets
+  vcn_id       = module.vcn.vcn_id
+  nat_route_id = module.vcn.nat_route_id
+
+  create_fss     = var.create_fss
```

Comment:
passing this parameter is not needed since you're making the entire module conditional.

Reply:
done